docs/content/languages/nodejs.md

```diff
@@ -0,0 +1,420 @@
+---
+title: "Fission functions with Nodejs"
+weight: 10
+---
+
+Fission supports functions written in Nodejs. Current fission nodejs runtime environment supports node version greater than 7.6.0. In this usage guide we'll cover how to use this environment, write functions, and work with dependencies.
+
+## Before you start
+
+We'll assume you have Fission and Kubernetes setup. If not, head over to the [install guide](../installation/_index.en.md).  Verify your Fission setup with:
+
+```
```

Comment:
Better to add type of code (here and other places) for syntax highlight.
```
```bash

```js
```

and `$` at the start of command will be greate